js/ui/collection/ui.collection_widget.base.js

```diff
@@ -48,6 +53,16 @@ var FOCUS_UP = "up",
     FOCUS_LAST = "last",
     FOCUS_FIRST = "first";
 
+
+function getPlainItems(groupedItems, group) {
+    let groupLevel = Array.isArray(group) ? group.length : 1,
+        getPlainItemsWithLevel = function(groupedItems, level) {
+            var plainItems = groupedItems.map(item => level > 0 ? getPlainItemsWithLevel(item.items, level - 1) : item);
+            return [].concat.apply([], plainItems);
+        };
+    return getPlainItemsWithLevel(groupedItems, groupLevel);
+}
+
```

Comment:
This function already exists in collection widget strategies. Don't make it universal on the base level. Use strategies please